pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -275,7 +275,7 @@ func (r *MXJobReconciler) ControllerName() string {
 }
 
 func (r *MXJobReconciler) GetAPIGroupVersionKind() schema.GroupVersionKind {
-	return mxjobv1.GroupVersion.WithKind(mxjobv1.Kind)
+	return mxjobv1.GroupVersion.WithKind("MXJob")
```

Comment:
Maybe we can add all `Kind` and `Singular` to [constants](https://github.com/kubeflow/tf-operator/blob/7ddf016a951fc79ce8239eb748312b86dfba0ebb/pkg/common/constants.go) file ?

Reply:
Sounds good. I remove previous constants in `register.go` to make them clean but didn't put them in constants.go. I will make the change